python_files/spark_with_baseball_data.py

```diff
@@ -0,0 +1,97 @@
+# Using Spark With Baseball DB Assignment 3
+# Will McGrath
+# October 3, 2020
```

Comment:
I like to add an est run time so others know if something is wrong when they run

Reply:
Ok I might try that next time thanks!